src/VisualStudio/Core/Def/ExternalAccess/VSTypeScript/Api/VSTypeScriptContainedDocumentWrapper.cs

```diff
@@ -35,14 +37,21 @@ public static bool TryGetContainedDocument(DocumentId documentId, out VSTypeScri
             return false;
         }
 
+        public void Dispose()
+            => _underlyingObject.Dispose();
+
+        public ITextBuffer SubjectBuffer
+            => _underlyingObject.SubjectBuffer;
+
+        public IVsContainedLanguageHost Host
+            => _underlyingObject.ContainedLanguageHost;
+
+        [Obsolete("Remove once TypeScript has stopped using this.")]
         internal AbstractProject Project
```

Comment:
What is still using this?

Reply:
Ah I think I get it now: this is no longer being used _after_ your TypeScript-side PR is merged.

Reply:
Precisely - I'm avoiding API breaks, but only for now. :)